lib/fake_braspag/sales.rb

```diff
@@ -40,6 +40,16 @@ class Sales < Sinatra::Base
       end
     end
 
+    put '/:PaymentId/conciliate' do
+      order = Order.find(params[:PaymentId])
+      if order.present? && order.PaymentMethod == 'Boleto' && ResponseToggler.enabled?('conciliate')
```

Comment:
Maybe we can move `ResponseToggler.enabled?('conciliate')` to `if` beginning to shortcut this validation when  `ResponseToggler.enabled` is false. wdyt?